### PR TITLE
UCP and Wallet sample

### PR DESCRIPTION
**Pull request by andytael**
UCP and Wallet sample

### CODE REVIEW

File: database/starters/oracle-spring-boot-starter-samples/oracle-spring-boot-sample-wallet/README.md [hunk 1]
```diff
@@ -0,0 +1,68 @@
+# Oracle Spring Boot Sample UCP using Wallet
+
+This sample application demonstrates how to connect to an Autonomous database using the Oracle Spring Boot Starter UCP, [UCP Documentation](https://docs.oracle.com/en/database/oracle/oracle-database/23/jjucp/), and Oracle Spring Boot Starter Wallet, [Database Connection with Wallet](https://docs.oracle.com/en/cloud/paas/autonomous-database/serverless/adbsb/connect-download-wallet.html), all while using the best connection pooling library for Oracle Database with Spring Boot.
+
+## Wallet
+
+1. Download the Wallet using the following [instructions](https://docs.oracle.com/en/cloud/paas/autonomous-database/serverless/adbsb/connect-download-wallet.html#GUID-DED75E69-C303-409D-9128-5E10ADD47A35).
+1. Unzip the Wallet ZIP file
+1. OPen the `sqlnet.ora` file and modify the line `WALLET_LOCATION = (SOURCE = (METHOD = file) (METHOD_DATA = (DIRECTORY="<DIRECTORY_OF_UNZIPPED_WALLET>")))` to make the `DIRECTORY` parameter point to the directory where you unzipped the wallet.
```
Comment (anders-swanson) [inline database/starters/oracle-spring-boot-starter-samples/oracle-spring-boot-sample-wallet/README.md:9]: OPen -> Open

Comment (andytael) [reply database/starters/oracle-spring-boot-starter-samples/oracle-spring-boot-sample-wallet/README.md]: Fixed

File: database/starters/oracle-spring-boot-starter-samples/oracle-spring-boot-sample-wallet/pom.xml [hunk 2]
```diff
@@ -0,0 +1,76 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
```
Comment (anders-swanson) [inline database/starters/oracle-spring-boot-starter-samples/oracle-spring-boot-sample-wallet/pom.xml:2]: copyright

File: database/starters/oracle-spring-boot-starter-samples/oracle-spring-boot-sample-wallet/src/main/java/com/oracle/database/spring/oraclespringbootsamplewallet/OracleSpringBootSampleWalletApplication.java [hunk 3]
```diff
@@ -0,0 +1,54 @@
+package com.oracle.database.spring.oraclespringbootsamplewallet;
```
Comment (anders-swanson) [inline database/starters/oracle-spring-boot-starter-samples/oracle-spring-boot-sample-wallet/src/main/java/com/oracle/database/spring/oraclespringbootsamplewallet/OracleSpringBootSampleWalletApplication.java:1]: copyright

**Review by anders-swanson — APPROVED** [review]

**Pull request merged**